js/plugins.js

```diff
@@ -317,10 +317,10 @@ plugins.factory('userPlugins', function() {
     });
 
     /*
-     * audio Preview
+     * Audio Preview
      */
     var audioPlugin = new UrlPlugin('audio', function(url) {
-        if (url.match(/\.(mp3|ogg|wav)\b/i)) {
+        if (url.match(/\.(flac|m3u|m4a|mid|midi|mp3|oga|ogg|opus|pls|wav|wma)\b/i)) {
```

Comment:
m3u is just a playlist container, could also be video: https://en.wikipedia.org/wiki/M3U :P